index.html

```diff
@@ -1,14 +1,67 @@
+        title="Pic3" style="width: 200px; height: 250px;" />

```

Comment:
You could have put a `class="image"` attribute onto these sections and given the CSS into that. Doing it that way would cut down the amount of code that you would have to write in your HTML and you would only have to define the CSS once in your CSS file